app/views/admin/api_docs/base/swagger.html.erb

```diff
@@ -31,10 +31,10 @@
 <% elsif @api_docs_service.specification.swagger_2_0? -%>
 
   <% content_for :javascripts do -%>
-    <%= javascript_include_tag "/_cdn_assets_/swagger-ui/2.2.10/swagger-ui.js" -%>
+    <%= javascript_include_tag "/dev-portal-assets/swagger-ui/2.2.10/swagger-ui.js" -%>
```

Comment:
shouldn't we use `cdn_asset ...` here?

Reply:
No because `cdn_asset` is a liquid tag and can only be used in liquid templates.